.github/workflows/release.yaml

```diff
@@ -28,7 +26,7 @@ jobs:
         env:
           CR_TOKEN: "${{ secrets.CR_TOKEN }}"
 
-      - name: Update Pages site
-        run: .ci/github/update-pages-site.sh
+      - name: Run hack/ci/github/steps/update-page.sh
```

Comment:
Here too, what doesn this script do?

Reply:
It runs the update-page step on our github ci solution. Isn't that all in the path/filename?